documentation/models.md

```diff
@@ -0,0 +1,141 @@
+<a name="collections"></a>
+## Collections
+### Rules
+The collection 'rules' stores information about the rules  to be sent to the core. It includes the VisualRule passed in
+by the Portal for giving it back when the Portal requests it
+
+Fields:
+* **_id** *ObjectId*: unique object id used by mongoDB
+* **name**  *string*: name of the rule
+* **service** *string* : service which the rule belongs to.
+* **subservice** *string*: subservice which the rule belongs to.
+* **tex** *string*: EPL sentence for the rule, to be propagated to core
+* **action** *object*: action to be executed when the rule is fired. Each action type has different field set as described in [Plain rules](plain_rules.md#actions)
+    * **type** ( *string* ): type of action.
```

Comment:
They are documented in [Plain rules](plain_rules.md#actions) as pointed out in doc. Not all actions have all the fields, but we can add a brief reference of those fields if you want